core/src/main/java/org/testcontainers/images/builder/ImageFromDockerfile.java

```diff
@@ -156,8 +170,24 @@ public ImageFromDockerfile withBuildArgs(final Map<String, String> args) {
         return this;
     }
 
+    /**
+     * Sets the Dockerfile to be used for this image. 
+     * @deprecated It is recommended to use {@link #withDockerfile} instead because it honors 
```

Comment:
Could you please add `@Deprecated` to the method?

Reply:
good catch! My IDE (Eclipse) crossed out the method just by having `@deprecated` in the javadoc